app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -64,7 +64,8 @@ input CustomerInput {
     dob: String @doc(description: "The customer's date of birth")
     taxvat: String @doc(description: "The customer's Tax/VAT number (for corporate customers)")
     gender: Int @doc(description: "The customer's gender(Male - 1, Female - 2)")
-    password: String @doc(description: "The customer's password")
+    password: String @doc(description: "The new customer's password")
+    currentPassword: String @doc(description: "The current customer's password")
```

Comment:
Please introduce new argument in the `updateCustomer` mutation, not the new field in the `CustomerInput`.
As per ticket AC:
> Introduce new optional argument for updateCustomer called currentPassword: String

It means

Reply:
Resolved by d3e5082 commit